src/provider/AccountImport.ts

```diff
@@ -1,13 +1,22 @@
+/**
+ * This interface represents the request object that is sent to the content script.
+ */
 export interface IAccountImportRequest {
   privateKey: string;
   publicKey: string;
 }
 
+/**
+ * This interface represents the response object that is sent by the content script.
```

Comment:
* This interface represents the response object sent by the content script,
 * indicating the result of an account import request.